### PR TITLE
Detach a line from 3 lines (attach new line on line reverse action)

### PR DESCRIPTION
**Pull request by FranckLecuyer**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:
this PR must be merged after PR : https://github.com/powsybl/powsybl-core/pull/2228

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/DetachLineFromLines.java [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+     */
+    public DetachLineFromLines(String lineAZId, String lineBZId, String lineCZId, String lineId, String lineName) {
+        this.lineAZId = Objects.requireNonNull(lineAZId);
+        this.lineBZId = Objects.requireNonNull(lineBZId);
+        this.lineCZId = Objects.requireNonNull(lineCZId);
+        this.lineId = Objects.requireNonNull(lineId);
+        this.lineName = lineName;
+    }
+
+    public DetachLineFromLines setLineAZId(String lineAZId) {
```
Comment (SlimaneAmar) [inline iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/DetachLineFromLines.java:67]: Why getters ans setters ?

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/DetachLineFromLines.java (hunk 1 above)
Comment (FranckLecuyer) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/DetachLineFromLines.java:67]: They are tested in testConstructor and testSetters tests

**Review by miovd — COMMENTED** [review]
A small comment about naming but otherwise okay for me!

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/RevertCreateLineOnLine.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+public class RevertCreateLineOnLine extends AbstractNetworkModification {
+
+    private static final Logger LOG = LoggerFactory.getLogger(RevertCreateLineOnLine.class);
+
+    private String lineAZId;
+    private String lineBZId;
+    private String lineCZId;
+
+    private String lineId;
+    private String lineName;
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/RevertCreateLineOnLine.java]: Done

What about the limits ? Same as what was done in RevertConnectVoltageLevelOnLine with the mergeLimits function?

[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/RevertCreateLineOnLine.java]: Done

**Review by miovd — APPROVED** [review]

**Pull request merged**